colibre/auto_plotter/snia_rates.yml

```
@@ -346,15 +348,56 @@ star_formation_rates_snia_rates_per_stellar_mass_50_Mstar5e10:
       units: "Solar_Mass/year"
   metadata:
     title: SFR-SNIa rate per stellar mass relation (50 kpc aperture)
-    caption: Uses a 50 kpc 3D aperture, only select galaxies with a stellar mass above 5e10 Msun.
-    section: SNIa Rate
+    caption: Uses a 50 kpc 3D aperture, only select galaxies with a stellar mass above 1e10 Msun.
```

Can you please use latex format here (and in other please), i.e. $10^{10} \mathrm{M_\odot}$ instead of 1e10?
```suggestion
    caption: Uses a 50 kpc 3D aperture, only select galaxies with a stellar mass above $10^{10} \mathrm{M_\odot}$.
```